app/src/main/java/edu/uc/coffeens/stocktracker/ui/main/MainFragment.kt

```diff
@@ -11,29 +11,15 @@ import androidx.lifecycle.ViewModelProvider
 import edu.uc.coffeens.stocktracker.R
 import kotlinx.android.synthetic.main.main_fragment.*
 
-class MainFragment : Fragment() {
-
-    companion object {
-        fun newInstance() = MainFragment()
-    }
+class MainFragment : Fragment(R.layout.main_fragment) {
 
     private lateinit var viewModel: MainViewModel
 
     /**
-     * This is what to do when the View is rendered.
+     *  This is what happens when the activity is rendered.
      */
-    override fun onCreateView(
-        inflater: LayoutInflater, container: ViewGroup?,
-        savedInstanceState: Bundle?
-    ): View {
-        return inflater.inflate(R.layout.main_fragment, container, false)
-    }
-
-    /**
-     *  This is what happened when the activity is rendered.
-     */
-    override fun onActivityCreated(savedInstanceState: Bundle?) {
-        super.onActivityCreated(savedInstanceState)
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
```

Comment:
onActivityCreated is deprecated + replaced onCreateView and onActivityCreated with just onViewCreated to reduce clutter https://developer.android.com/jetpack/androidx/releases/fragment#1.3.0-alpha02